rules/python/django/csrf_protection_disabled.yml

```diff
@@ -0,0 +1,32 @@
+imports:
+  - python_shared_lang_import4
+patterns:
+  - pattern: $<CSRF_EXEMPT>
+    filters:
+      - variable: CSRF_EXEMPT
+        regex: \A@(django\.)?(views\.)?(decorators\.)?(csrf\.)?csrf_exempt\z
```

Comment:
I think a decorator is just a function call/expression. So you should be able to use an import rule here, though it may require some additional engine support.

Reply:
Will merge for now and  come back to this one -- I couldn't get the import working with the `@`